src/Parse/Parse.hs

```diff
@@ -273,9 +281,25 @@ parseTerm5 = parsePeek >>= \ t -> case t of
     parsePeek >>= \ t -> case t of
         TkRangle -> pure (UsProd Additive [])
         _ -> pure (UsProd Additive) <*> (parseTerm1 >>= \ tm -> parseDelim parseTerm1 TkComma [tm])) <* parseDrop TkRangle
+  TkNat n -> (
+    parsePeeks 2 >>= \t1t2 -> case t1t2 of
```

Comment:
Adding addition seems like a great idea but I think we should do it completely. Let's take this out for now.